.travis.yml

```diff
@@ -22,6 +22,13 @@ env:
   - secure: XpyyEMPAvqtQIXap/dLAgl8JhAKfoSdsmoGY659eHZcijYv6d8pOX7JPUfwHhteoso/MkX9o+F8t7z7AIigNi0FJHQedP4cwhtbYEDFr8w4tRz9TYf7u4d/UUosVD4x95v9Yp+G84P/Rw6Vr90TgXExo4hVxJx2faNp9xkpMqE8=
   #SAUCE_ACCESS_KEY=********-****-****-****-************(irdan/privly-applications)
   - secure: gCbEFk/4uWMa41BFLLwHaoJebbZHpxIF6H2NCqiAYlVwAKB2Dzya9imwKzm6XUCXRyUakN8eaJBO9d9OQtRV+D2Uap7sUDGoAjeGgKQuGYpbSYgN8NQVAJdY+WeAeH4Ul4ExZOJKQH7AL6pFkCHn9m/RnXCosrOG+ob4fievJ8s=
+  #SAUCE_USERNAME=jatindhankhar(jatindhankhar/privly-applications)
+  - secure: kCjnbIfAAbSwAOZISCzsxsoI1DjDLPXzLwcDMCDSluJypfyxqoOrwBBs+u96wwHylr+lsYzEdAa+A7BWjlSzkYUwkCMRUAlu2+5fXCjelG4uGUFio+AiipF8HPCqgsxA9e5i+JEGk/aWkt6i3obZP5sQnaLMOigYA3ngEce3yp4=
+  #SAUCE_ACCESS_KEY=********-****-****-****-************(jatindhankhar/privly-applications)
+  - secure: BOfaPuGkeDJX+YhpoZn7zXzZLs+Kb6GALuap7jRX33M6igamx+MCMl1OTtKRuFiosZBDZQrs/E24QzRzSF9gBzqSqPTgQNUkoECwaIm0IIsdA8ff1SoFuFw8x8iFKFPXVb5C72xtwqXfqRAqchX6Jd4v/Wh+mEDMESxE5S5XIRY=
+  #SAUCE_URL=http://jatindhankhar:********-****-****-****-************@ondemand.saucelabs.com/wd/hub
+  - secure: sWkxx3v8RxChO0lZSYqNjkYvfg1Re6LwpDeBZIgw1qBBBg5UdCLJZitOAAqKsdoUpeVD0Hk9o6UdWfVnYAw81xArvHpl32VQgItY60UVhHVoszX5CQFo7bp0p59dPZbzx4XjJfZhM0btrHOUHpJkeoU6Ia9Z+gTiZ04uC+Jqi/s=
+  #CODECLIMATE_REPO_TOKEN=**************************(smcgregor/privly-applications)
```

Comment:
Just did that
Waiting for the build to yield green on both ends
I have a little question, 
Travis builds will eat up my monthly free minutes?
Concerned because if quota is out, tests may fail